schema/deploy/shipping/views.sql

```diff
@@ -669,4 +669,39 @@ comment on view shipping.sample_with_best_available_encounter_data_v1 is
     'Version 1 of view of warehoused samples and their best available encounter date and site details important for metrics calculations';
 
 
+create or replace view shipping.return_results_v2 as
+
+    select barcode,
+           case
+             when sample_id is null then 'notReceived'
+             when sample_id is not null and count(present) = 0 then 'processing'
+             when count(present) > 0 then 'complete'
+           end as status,
+           array_agg(distinct organism::text)
+            filter (where present and organism is not null)  as organisms_present,
+           array_agg(distinct organism::text)
+            filter (where not present and organism is not null) as organisms_absent
+
+      from warehouse.identifier
+      join warehouse.identifier_set using (identifier_set_id)
+      left join warehouse.sample on uuid::text = sample.collection_identifier
+      left join warehouse.encounter using (encounter_id)
+      left join shipping.presence_absence_result_v2 on sample.identifier = presence_absence_result_v2.sample
+
+    --These are all past and current collection identifier sets not including self-test
+    where identifier_set.name in ('collections-kiosks',
+                                  'collections-swab&send')
+      and (organism is null or
+          -- We only return results for COVID-19, so omit all other presence/absence results
+          organism <@ '{"Human_coronavirus.2019"}'::ltree[])
+          -- We only want results collected after Izzy updated the REDCap consent form in swab & send.
+          -- This filter-by timestamp comes from REDCap
+      and coalesce(encountered, date_or_null(warehouse.sample.details->>'date')) > '2020-03-04 08:35:00-8'::date
```

Comment:
The cast of the timestamp to `::date` means that the time part is ignored (i.e. treated as 00:00:00). Changing to `::timestamp with time zone` or using `2020-03-05` (one day later) would both be acceptable.

Reply:
Resolved